### DISCUSSION TITLE
how to  print the value of the `MCU::BANK_RIGHT`/`BANK_LEFT` button

### DISCUSSION THREAD

**Discussion by pukulsesuatu** [tttapa/Control-Surface#791] [Q&A] [answered]
hello, I want to ask how to display and print the value of the `MCU::BANK_RIGHT`/`BANK_LEFT` button. Like using the IncrementDecrementSelector with the `.get()` command to print and display the button value, so we know what the current bank number is. Thanks, I hope someone can answer that.

**Comment by tttapa** [comment, marked as answer]
The Arduino doesn't know the value, only the DAW knows it. The Arduino just sends left/right, it doesn't know the absolute value.

If you're lucky, the DAW might send it to you as a response (see https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/d85/Mackie-Control-Universal-Reverse-Engineering_8ino-example.html), or you might be able to infer it some other way.  
Perhaps you can send e.g. 4 left messages, so you know that the leftmost bank has to be selected.

**Reply by pukulsesuatu** [reply]
Thanks for information,but I am not understand with another way like tou said to send e.g 4 left message what you mean? Thanks
